build.gradle

```diff
@@ -1,34 +1,50 @@
-plugins {
-    id 'java'
-    id 'checkstyle'
-    id 'org.jetbrains.intellij' version '0.4.15'
-    id "org.sonarqube" version "2.8"
-}
-
-group 'fi.aalto.cs'
-version '0.1.' + System.getenv('TRAVIS_BUILD_NUMBER')
-
-sourceCompatibility = 1.8
-
-repositories {
-    mavenCentral()
-}
-
-dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-}
-
-intellij {
-    version '2019.3.1'
-    plugins = ['java']
-}
-
-publishPlugin {
-    token = System.getenv('INTELLIJ_PUBLISH_TOKEN')
-    channels = [System.getenv('INTELLIJ_PUBLISH_CHANNEL')]
-}
-
-checkstyle {
-    configFile = file('/google_checks.xml')
-    maxWarnings = 0
-}
+plugins {
+    id 'java'
+    id 'checkstyle'
+    id 'org.jetbrains.intellij' version '0.4.15'
+    id "org.sonarqube" version "2.8"
+    id 'jacoco'

```

Comment:
JaCoCo is back! Okay, this is here just so that I can create coverage reports locally. Travis CI is not involved. :P

Reply:
And Jacoco results are sent to Sonar.